src/styles/blocks/variables.scss

```
@@ -0,0 +1,5 @@
+$colorGrey: #616070;
+$color-skyBlue: #00acdc;
+$cardSize: 200px;
+$cardGap: 48px;
```

It's not JS, so name your variables in one convention - using `-` to separate words
```suggestion
$card-size: 200px;
$card-gap: 48px;
```